sigs.yaml

```diff
@@ -1992,7 +1992,7 @@ workinggroups:
     dir: wg-multitenancy
     mission_statement: >
       Define the models of multitenancy that Kubernetes will support. Discuss and execute upon any remaining work that needs to be done to support these models. Create conformance tests that will prove that these models can be built and used in production environments.
-    charter_link:
+    charter_link: https://docs.google.com/document/d/1SkVdOPR4jozYDT8ro51hU3yrf1sHS8Gez73xM3PCsVo/edit
```

Comment:
Charters should be in-repo, so they can be versioned and approved by steering.

Reply:
Yup, thought so too. There was one more wg which had a google-doc charter. Removed it as well. :+1: